### DISCUSSION TITLE
mp3 and wav files in TTS data preparation

### DISCUSSION THREAD

**Discussion by nellorebhanuteja** [espnet/espnet#4646] [Q&A] [answered]
Hi

I just noticed, in the IndicSpeech TTS recipe, we're creating both [wav and mp3 files](https://github.com/espnet/espnet/blob/a7bd6522b32ec6472c13f6a2289dcdff4a846c12/egs2/indic_speech/tts1/local/data_prep.py#L36) in data preparation stage.
what is the reason for this ?

**Comment by kan-bayashi** [comment]
@peter-yh-wu may be able to answer.

**Comment by peter-yh-wu** [comment, marked as answer]
The mp3 files were used in listening tests, but they are optional for training
